File: 01-JavaScript/11-guess-number-person.js
const prompt = require("prompt-sync")({ sigint: true });
/* We are guessing which number the computer is thinking of. */

console.log("I am thinking of a number between 0 and 5.");

const randomNumber = Math.floor(Math.random() * 6);

let guessNumber = parseInt(prompt("Take a guess: "));

let guessCount = 1;

if (guessNumber !== randomNumber) {
    while (guessNumber !== randomNumber) {
        guessNumber = parseInt(prompt("Nope! Take a new guess: "));
        guessCount += 1;
    }
    console.log(`You got it! It was number: ${randomNumber}. It has taken you ${guessCount} guesses.`);

} else {
    console.log("You got it from the first attempt!");
}

File: 02-Python/11-guess-number-person.py
#!/usr/bin/python

#  We are guessing which number the computer is thinking of.

while True:

    import random

    print("I am thinking of a number between 0 and 5.")

    random_number = random.randint(0, 5)

    guess_number = int(input("Take a guess: "))

    guess_count = 1

    if guess_number != random_number:

        while guess_number != random_number:
            guess_number = int(input("Nope! Take a new guess: "))
            guess_count += 1

        else: 
            print("You got it! It was number: " + str(random_number) + "." + " It has taken you " + str(guess_count) + " guesses!")

    else:
        print("You got it from the first attempt!")


    if input("Continue? Y/N: ").lower() not in {"y", "yes"}:
        break